config/manifests/bases/cinder-operator.clusterserviceversion.yaml

```diff
@@ -54,8 +54,9 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - openstack
-  - cn-openstack
+  - OpenStack
+  - Volume
```

Comment:
Wouldn't it be better to use `Block Storage` instead of `Volume`?

Reply:
Ah, yes. I was referring to service type but Block Storage does sound better.